cmd/runhcs/container.go

```diff
@@ -572,10 +566,12 @@ func startContainerShim(c *container, pidFile, logFile string) error {
 	if err != nil {
 		return err
 	}
-	defer shim.Release()
+	defer func() {
+		_ = shim.Release()
```

Comment:
This is obnoxious. Maybe we need to think about turning off the error check lint in the future.

Reply:
I think it's valuable to have on. I think we should just find a better way to handle cases where we don't really care about the error. Maybe log a warning? Probably depends on the case though.